Gini-iOS-SDK/GINIDocumentTaskManager.m

```diff
@@ -17,18 +17,18 @@
     return [originalTask continueWithBlock:^id(BFTask *task) {
         if (task.error && [task.error.domain isEqualToString:NSURLErrorDomain]) {
             switch(task.error.code) {
-                // HTTP #404
+                    // HTTP #404
```

Comment:
Why did you add this tab? Shouldn't it align with the case statement?

Reply:
I don't know why Xcode did this, but I can change it.